keyboards/minimacro5/keymaps/default/keymap.c

```diff
@@ -0,0 +1,44 @@
+#include QMK_KEYBOARD_H
+
+#define _MAIN 0
```

Comment:
Just a nit pick, but we prefer enums rather than defines.  They basically work the same, but it's easier to maintain long term. 

```suggestion
enum layers (
     _MAIN,
};
```